src/renderer/atlas/shader_ps.hlsl

```diff
@@ -147,7 +147,8 @@ float4 main(float4 pos: SV_Position): SV_Target
                 // See DWrite_GrayscaleBlend
                 float intensity = DWrite_CalcColorIntensity(foregroundStraight);
                 float contrasted = DWrite_EnhanceContrast(glyph.a, blendEnhancedContrast);
-                color = fg * DWrite_ApplyAlphaCorrection(contrasted, intensity, gammaRatios);
+                float4 alphaCorrected = DWrite_ApplyAlphaCorrection(contrasted, intensity, gammaRatios);
+                color = alphaBlendPremultiplied(color, alphaCorrected * fg);

```

Comment:
Here's a direct link showing the old code using `alphaBlendPremultiplied` here: https://github.com/microsoft/terminal/commit/5964060b403842a34dee45c15f186e6c04227777#diff-ca02942b22b9baa16bcef607b524e1e4590793109f302c097b6705f0d1383310L135